test-files/useMessageActions.dynamic.tsx

```
@@ -0,0 +1,14 @@
+// This file should be allowed to import from another dynamic file
+import { useMessage } from './MessageContext.dynamic';
+
+export const useMessageActions = () => {
+  const { message } = useMessage();
+
+  const sendMessage = () => {
+    console.log(`Sending message: ${message}`);
+  };
+
+  return {
+    sendMessage
```

ESLint

[error] Insert `,`

(prettier/prettier)